tests/test_s3/test_s3_multipart.py

```diff
@@ -148,6 +148,47 @@ def test_multipart_upload():
     response["Body"].read().should.equal(part1 + part2)
 
 
+@mock_s3
+@reduced_min_part_size
+def test_multipart_upload_with_encoded_char():
```

Comment:
This is the same test as `test_multipart_upload`, right? 
I think it's cleaner to just parametrize that test, using `pytest.mark.parametrize`

There are other tests in this file using the same fixture, so you can use those as a guide. Let me know if you run into any issues.

Reply:
Makes sense. Done.